### DISCUSSION TITLE
A prop for ConnecctButton to always show the current chain even if there is only "one" chain

### DISCUSSION THREAD

**Discussion by ITZSHOAIB** [rainbow-me/rainbowkit#1780] [Ideas]
Issue: https://github.com/rainbow-me/rainbowkit/issues/1418

Regarding this issue, what approach would be suitable?
- Adding an option to the exiting chainStatus prop to achieve this.
- Adding a whole new prop to show the chainStatus all the time
- By updating the condition `chain && (chains.length > 1 || unsupportedChain)`
- Or something else...

Please feel free to share your thoughts.

**Comment by ITZSHOAIB** [comment]
@KosmosKey any thoughts?

**Reply by 0xmkh** [reply]
Great idea! I think i wanna do exactly like that, but instead i'll introduce a prop called `showCurrentChain` and based off of that we will determine whether we should show a current chain without having to open the chain modal.

**Reply by ITZSHOAIB** [reply]
But wouldn't this be a redundant prop as we already have a prop called `chainStatus`. We can add another enum value to this prop to achieve this. Feel free to share your thoughts as well.

**Reply by 0xmkh** [reply]
`chainStatus` prop is mainly used to hide avatar and name. This wouldn't apply for "always show current chain" feature. I've created a PR here which you can checkout: https://github.com/rainbow-me/rainbowkit/pull/1793
